index.html

```diff
@@ -19,21 +19,29 @@
         <a class="logo" href="#">
             Mahmoud Makhlouf
         </a>
-
-        <a href="#">
+        <ul class="navbar-links">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+            <li><a href="#"><img src="img/icon-mail.png" alt="mail"></a></li>
+        </ul>
+        <a class="toggle-button" href="#">
             <img src="img/Union.png" alt="collapsed-menu">
         </a>
 
     </header>
 
     <div class="headline">
-        <h1 class="name">Hey There. I’m Mahmoud</h1>
-        <h2 class="job-title">I’am a Software Developer</h2>
-        <p>I can help you build a product ,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.
-        </p>
+        <div class="intro">
+            <h1 class="name">Hey There.<br>I’m Mahmoud</h1>
+            <h2 class="job-title">I’am a Software Developer</h2>
+            <p>I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        
```

Comment:
- [ ] Good job setting up your desktop version! However, your design needs some adjustments in order to look similar to the Figma template:

![Captura de pantalla 2022-03-25 a la(s) 7 42 02 p  m](https://user-images.githubusercontent.com/83717285/160218512-e66c57c1-729f-41f5-be7c-1a9412c17646.png)

Try adding some space between the text and the social icons and making your paragraph narrower.